### PR TITLE
enable disable policy api doc added

### PR DESCRIPTION
**Pull request by alamirfan**
POST /admin/expense_policy_rules/enable
POST /admin/expense_policy_rules/disable

Request payload
```
{
  "data": {
    "id": "tprXsN9xJXG4W"
  }
}
```

Response
```
{
  "data": {
    "id": "tpr12345xyz",
    "description": "Receipt is mandatory for expense above $50",
    "source": "WEBAPP_POLICY_FORM",
    "expense_limit_type": "MONTHLY",
    "expense_currency": "USD",
    "expense_amount_max": 10000,
    "expense_amount_min": 500,
    "expense_distance_max": 10000,
    "expense_distance_min": 500,
    "expense_category_ids": [
      2222,
      1221
    ],
    "expense_category_ids_op": "IN",
    "expense_project_ids": [
      1234,
      5678
    ],
    "expense_project_ids_op": "IN",
    "expense_cost_center_ids": [
      1987,
      3456
    ],
    "expense_cost_center_ids_op": "IN",
    "expense_exchange_rate_difference_percentage_allowed": 10,
    "expense_distance_difference_percentage_allowed": 8,
    "expense_custom_fields": [
      {
        "name": "cf1",
        "value": "string1"
      }
    ],
    "expense_custom_fields_op": {
      "field_name": "string"
    },
    "expense_locations": [
      {
        "city": "London",
        "state": "London",
        "display_name": "string",
        "country": "United Kingdom",
        "formatted_address": "221 Baker St, Marylebone, London, United Kingdom",
        "latitude": 12.971599,
        "longitude": 77.594566
      }
    ],
    "expense_locations_op": "IN",
    "expense_creation_delay_days_max": 12,
    "expense_on_holiday_allowed": true,
    "expense_without_receipt_allowed": true,
    "expense_spent_at_max": "2020-06-01T01:18:19.292-08:00",
    "expense_spent_at_min": "2020-06-01T01:18:19.292-08:00",
    "expense_payment_modes": "PERSONAL_CASH_ACCOUNT",
    "expense_payment_modes_op": "IN",
    "expense_without_ccc_match_allowed": true,
    "expense_include_foreign_currency": true,
    "expense_flight_travel_class": "ECONOMY",
    "expense_train_travel_class": "STANDARD",
    "fiscal_year_start_month": 4,
    "expense_vehicle_type": "ELECTRIC_CAR",
    "expense_annual_distance_max": 1200,
    "employee_level_ids": [
      "lvl1sdfad",
      "lvl2sdfas"
    ],
    "employee_level_ids_op": "IN",
    "employee_custom_fields": [
      {
        "name": "cf1",
        "value": "string1"
      }
    ],
    "employee_custom_fields_op": {
      "field_name": "string"
    },
    "employee_department_ids": [
      "dept1asdfa",
      "dept2adfad"
    ],
    "employee_department_id_op": "IN",
    "employee_locations": [
      {
        "city": "London",
        "state": "London",
        "display_name": "string",
        "country": "United Kingdom",
        "formatted_address": "221 Baker St, Marylebone, London, United Kingdom",
        "latitude": 12.971599,
        "longitude": 77.594566
      }
    ],
    "employee_locations_op": "IN",
    "employee_user_ids": [
      "usabcds1",
      "usabcds2"
    ],
    "employee_user_ids_op": "IN",
    "employee_titles": [
      "CEO",
      "Manager"
    ],
    "employee_titles_op": "IN",
    "action_flag": true,
    "action_show_warning": true,
    "action_make_unreportable": true,
    "action_remove_employee_approver1": true,
    "action_add_employee_approver2": true,
    "action_add_employee_approver3": true,
    "action_add_project_approver1": true,
    "action_add_project_approver2": true,
    "action_add_department_head_approver": true,
    "action_cap_amount": true,
    "action_add_approver_user_ids": [
      "usabcds3",
      "usabcds4"
    ],
    "action_override_distance_slab_rate": 20
  }
}
```

### CODE REVIEW

**Review by viiicky — APPROVED** [review]

File: reference/admin.yaml [hunk 1]
```diff
@@ -10428,3 +10439,113 @@ paths:
...
+        content:
+          application/json:
+            schema:
+              type: object
+              additionalProperties: false
+              required:
+                - data
+              properties:
+                data:
+                  $ref: '#/components/schemas/policy_id_in'
```
Comment (aditya999123) [inline reference/admin.yaml:10461]: expense_policy_enable_in

File: reference/admin.yaml [hunk 2]
```diff
@@ -7038,6 +7038,17 @@ components:
         action_override_distance_slab_rate:
           type: number
           example: 20
+    policy_id_in:
```
Comment (aditya999123) [inline reference/admin.yaml:7041]: expense_policy_enable_in

File: reference/admin.yaml [hunk 3]
```diff
@@ -7038,6 +7038,17 @@ components:
...
+      type: object
+      required:
```
Comment (aditya999123) [inline reference/admin.yaml:7043]: additionalPropoerties: False

File: reference/admin.yaml [hunk 4]
```diff
@@ -10428,3 +10439,113 @@ paths:
...
+        content:
+          application/json:
+            schema:
+              type: object
+              additionalProperties: false
+              required:
+                - data
+              properties:
+                data:
+                  $ref: '#/components/schemas/policy_id_in'
```
Comment (aditya999123) [inline reference/admin.yaml:10516]: expense_policy_disable_in

File: src/components/schemas/expense_policies.yaml [hunk 5]
```diff
@@ -504,4 +504,16 @@ policy_out:
...
\ No newline at end of file
+      example: 20
+
+policy_id_in:
+  type: object
+  required:
+    - id
+  description: |
+    This represents the id of the policy.
+  properties:
```
Comment (aditya999123) [inline src/components/schemas/expense_policies.yaml:515]: additionalproperties false

**Review by viiicky — APPROVED** [review]

**Pull request merged**